docs/orbs-authoring.md

```diff
@@ -3,6 +3,8 @@ This document largely covers the tooling and flow of authoring and publishing yo
 
 Orbs can be authored [inline](inline-orbs.md) in your config.yml file or authored separately and then published to to the orb registry for reuse across projects. This document is largely about how to publish orbs for use across projects. For inline orbs see the [Writing Inline Orbs](inline-orbs.md) document.
 
+**All published orbs can be read and used by anyone, not just members of your organization**. You will have to use inline orbs if you need to write orbs privately (and it is always best to put things that need to be secure into context/environment variables rather than config).
```

Comment:
Let's not talk about inline orbs as an answer to "private orbs" because it's not at all satisfying as an answer. (we are more simply saying we don't yet have private orbs). Please change this to say:

**Orbs are always world-readable.**. All published orbs can be read and used by anyone, not just members of your organization. In general, we strongly recommend that you do not put secrets or other sensitive variables into your configuration. Instead, use contexts or project environment variables and use the names of those environment variables in your orbs.